GVFS/GVFS.Common/Prefetch/Git/DiffHelper.cs

```diff
@@ -143,19 +143,20 @@ public void PerformDiff(string sourceTreeSha, string targetTreeSha)
                 metadata.Add("DirectoryOperationsCount", this.TotalDirectoryOperations);
                 metadata.Add("FileDeleteOperationsCount", this.TotalFileDeletes);
                 metadata.Add("RequiredBlobsCount", this.RequiredBlobs.Count);
+                metadata.Add("FileAddOperationsCount", this.FileAddOperations.Sum(kvp => kvp.Value.Count));

```

Comment:
This is just a bit of extra logging metadata, it looks like.

Reply:
👍  Yep; it was useful for me to have, and fit the theme of the other items being logged.